content/cloud/develop.md

```diff
@@ -492,15 +562,14 @@ If the build fails, check:
 
 ```bash
 $ spin build
-Building component hello-python with `spin py2wasm app -o app.wasm`
-Spin-compatible module built successfully
+Building component hello-python with `componentize-py -w spin-http componentize app -o app.wasm`
+Component built successfully
 Finished building all Spin components
 ```
 
 If the build fails, check:
 
-* Are you in the `hello_python` directory?
-* Did you install the `py2wasm` plugin?
+* Are you in the `hello-python` directory?
```

Comment:
Oh! The other thing to check is are you in the virtual environment. I'm not sure if that's the exact phrasing - Karthik or Joel will know though

(ETA: I see you already have this and other words in the quickstart - could copy those?)

Reply:
Good call; updated now.